pkg/kubelet/config/file.go

```diff
@@ -189,8 +189,7 @@ func (s *sourceFile) extractFromFile(filename string) (pod *v1.Pod, err error) {
 		return pod, nil
 	}
 
-	return pod, fmt.Errorf("%v: read '%v', but couldn't parse as pod(%v).\n",
-		filename, string(data), podErr)
+	return pod, fmt.Errorf("%v: couldn't parse as pod(%v), please check config file.\n", filename, podErr)
```

Comment:
That's fair. Fine to leave as-is in this PR then. 
Generally speaking, though, `config` is too ambiguous. It would be worth another PR to add clarity to these log messages IMO.